cmd/maya-exporter/app/collector/zvol/pool_synctime_test.go

```diff
@@ -37,9 +37,9 @@ func TestPoolSyncTimeMetricCollector(t *testing.T) {
 			zpoolLastSyncTimeOutput: `cstor-c6f17743-e5d7-11e9-b673-42010a800112      io.openebs:livenesstimestamp    1570625404      local
 			cstor-c6f17743-e5d7-11e9-b673-42010a800112/pvc-053647ca-e5d8-11e9-b673-42010a800112     io.openebs:livenesstimestamp    1570625404      inherited from cstor-c6f17743-e5d7-11e9-b673-42010a800112`,
 			expectedOutput: []string{
-				`openebs_zpool_last_sync_time{pool="cstor-c6f17743-e5d7-11e9-b673-42010a800112"} 1.570625404e\+09`,
-				`openebs_zpool_state_unknown{pool="cstor-c6f17743-e5d7-11e9-b673-42010a800112"} 0`,
-				`openebs_zpool_sync_time_command_error{pool="cstor-c6f17743-e5d7-11e9-b673-42010a800112"} 0`,
+				`openebs_zpool_last_sync_time{pool=""} 1.570625404e\+09`,
```

Comment:
can you modify this test to use the hostname?

Prior to running the test you can modify the os.evn hostname to what will typically be seen within the pool pod and then set it back to whatever was the original value after the test is done. 

Example: https://github.com/openebs/maya/blob/master/cmd/mayactl/app/command/volume_stats_test.go#L118